File: Scripts/newTask/classes/ruby/classes.rb
class MyClass

  def initialize
    @instance_var = 67712
  end

  def printVariable
    puts "Variable value is #{@instance_var}."
  end

end

for i in 0..1000000

my_class = MyClass.new #allocates an object and calls it's initialize method, then returns it.
my_class.printVariable

end

File: Scripts/newTask/classes/php/classes.php
<?php

class MyClass {
    public static $classVar;
    public $instanceVar; // can also initialize it here
    function __construct($value) {
        $this->instanceVar = $value;
    }
    function someMethod() {
        self::$classVar = 3;
	echo "Variable value is $this->instanceVar\n";
    }
}

for ( $i = 0; $i < 1000000; $i++) {
	$myObj = new MyClass(1974);
	$myObj->someMethod();
}
?>
